docs/content/en/docs/tasks/troubleshoot/troubleshooting.md

```diff
@@ -138,20 +125,148 @@ If you don't have any other docker containers running you may want to run `docke
+, try rerunning with —force-cleanup to force delete previously created bootstrap cluster
+```
+
+The cluster already exists, so you can delete the old one:
+
+```bash
+kind delete cluster --name cluster-name
+```
+
+### Creating new workload cluster hangs or fails
+
+Cluster creation appears to be hung waiting for the Control Plane to be ready.
+If the CLI is hung on this message for over 30 mins, something likely failed during the OS provisioning:
+
+```
+Waiting for Control Plane to be ready
+```
+
+Or if cluster creation times out on this step and fails with the following messages:
+
+```
+Support bundle archive created {"path": "support-bundle-2022-06-28T00_41_24.tar.gz"}
+Analyzing support bundle {"bundle": "CLUSTER_NAME/generated/bootstrap-cluster-2022-06-28T00:41:24Z-bundle.yaml", "archive": "support-bundle-2022-06-28T00_41_24.tar.gz"}
+Analysis output generated {"path": "CLUSTER_NAME/generated/bootstrap-cluster-2022-06-28T00:43:40Z-analysis.yaml"}
+collecting workload cluster diagnostics
+Error: waiting for workload cluster control plane to be ready: executing wait: error: timed out waiting for the condition on clusters/CLUSTER_NAME
+```
+In either of those cases, the following steps can help you determine the problem:
+
+1. Export the kind cluster’s kubeconfig file:
+
+    ```bash
+    export KUBECONFIG=${PWD}/${CLUSTER_NAME}/generated/${CLUSTER_NAME}.kind.kubeconfig
+    ```
+1. If you have provided BMC information:
+
+    * Check all of the machines that the EKS Anywhere CLI has picked up from the pool of hardware in the CSV file:
+
+        ```bash
+        kubectl get machines -A
+        ```
+    * Check if those nodes are powered on. If any of those nodes are not powered on after a while then it could be possible that BMC credentials are invalid. You can verify it by checking the logs:
+
+        ```bash
+        kubectl get bmt -n eksa-system
+        kubectl get bmt <bmt-name> -n eksa-system -o yaml
+        ```
+
+    BMC credentials are incorrect if the message in the bmt log says something like `Failed to connect to BMC: failed to open connection to BMC`. You might have enable “IPMI over LAN” for that machine if it says something like “failed to perform OneTimeBootDeviceAction”
+
+1. If the machine is powered on but you see `linuxkit is not running`, then Tinkerbell failed to serve the node via iPXE. In this case, you would want to:
+
+    * Confirm no other DHCP service responded to the request and check for any errors in the BMC console. 
+    * Check the boots service logs from the machine where you are running the CLI to see if it received and/or responded to the request:
+
+        ```bash 
+        docker logs boots
+        ```
+
+1. If you see `linuxkit is running*, click enter in the BMC console to access the linuxkit terminal. Run the following commands to check if the tink-worker is running.
+
+    ```bash
+    docker logs <container-id>
```

Comment:
Reversee the order of these commands as you need to run ps first to identify the container id for logs

Reply:
yeah, makes sense. @chrisnegus can you reverse the order of both the commands here?
### docker ps -a
docker logs <container-id>

Reply:
Done.